src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Image.cs

```diff
@@ -156,6 +156,9 @@ public async Task When_Transitive_Asset_Loaded()
 
 		[TestMethod]
 		[RunsOnUIThread]
+#if __MACOS__
+		[Ignore("Fails on macOS for resising and assets locations https://github.com/unoplatform/uno/issues/6261")]
```

Comment:
This is linking to https://github.com/unoplatform/uno/pull/6261

Is that the correct link?

Reply:
It's not, I will fix it in the next PR. Thanks!